src/services/admin/service.py

```
@@ -223,10 +223,17 @@ async def _ensure_tables(self) -> None:
                 assigned_at INTEGER NOT NULL,
+                mentee_login TEXT NOT NULL DEFAULT '',
                 PRIMARY KEY (org, issue_repo, issue_number)
             )
             """
         )
+        try:
+            await self._d1_run(
+                "ALTER TABLE mentor_assignments ADD COLUMN mentee_login TEXT NOT NULL DEFAULT ''"
+            )
+        except Exception:
+            pass
```

Ruff (0.15.6)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)